packages/app-store/_utils/useAddAppMutation.ts

```diff
@@ -40,16 +40,18 @@ function useAddAppMutation(_type: App["type"] | null, options?: Parameters<typeo
 
     const json = await res.json();
 
+    const externalUrl = !json.url.startsWith(window.location.origin);
```

Comment:
`json.url` can be a relative path like '/setup' and in that case it would be considered external URL wrongly.
We need to copy `/https?:\/\//.test(json.url)` logic here as well.